packages/odie-client/src/style.scss

```
@@ -38,6 +38,10 @@ $blueberry-blue: #3858e9;
 	overscroll-behavior: contain;
 	padding-top: 16px;
 
+	&.is-notice-visible {
```

I think we if do this, we don't need any other changes
```suggestion
	&:has(.odie-notice) {
```